plugins/context_datalayer_reaction.inc

```diff
@@ -22,36 +22,36 @@ class context_datalayer_reaction extends context_reaction {
       '#default_value' => isset($defaults['overwrite']) ? $defaults['overwrite'] : FALSE,
     );
 
-    $form['data'] = array(
+    $form['context_datalayer_reaction']['data'] = array(
```

Comment:
This looks really messy but it was the only way I could see to separate the two items.  So many instances of both things being fired/ the wrong one coming up/ IDs incrementing because they get repeated ... I just bit the bullet and decided to be explicit and it just worked thankfully.